README.md

```diff
@@ -2,27 +2,52 @@
 
 This repository contains a sample app in Swift using the [Whereby iOS SDK](https://github.com/whereby/ios-sdk).
 
-Other platforms: 
+Other platforms:
+
 - [Android SDK](https://github.com/whereby/android-sdk)
 - [Browser SDK](https://github.com/whereby/browser-sdk)
 
 ## Prerequisites
+
 - The latest stable version of [Xcode](https://apps.apple.com/us/app/xcode/id497799835)
 - The latest stable version of [cocoapods](https://cocoapods.org/)
 - Sign up to [Whereby Embedded](https://whereby.com/information/embedded/) account
 - [Create a room](https://docs.whereby.com/creating-and-deleting-rooms) in your Whereby Embedded account
 
-## Running the app
+## Using the app with Cocoapods
+
 1. Install WherebySDK dependency using cocoapods. Starting at the root folder of this repo:
+
 ```
-cd WherebySDKDemo
+cd ~/ios-sdk-demo-main/Demo-CocoaPods
```

Comment:
So technicaly, `~/ios-sdk-demo-main` depends on your machine. I believe might be a combination of the default folder name with the current branch but you could decide to name it as you wish. Maybe something like `cd [project root path]/Demo-CocoaPods` would be more generic.